api/src/cli/utils/create-env/env-stub.liquid

```diff
@@ -1,60 +1,331 @@
 ####################################################################################################
-## General
+# 
+# These values set variables which modify core settings of Directus. The variables can be stored
+# directly in your environment (which means the variables are accessible from the operating system)
+#
+# It is more performant to set these values in your environment directly rather than in this file
+# but can be left here whilst developing in case you want to change them. If you set all the 
+# variables directus needs directly in your environment, there is no need for this file.
+#
+# To set permanent environment variables, create an .sh file in the "/etc/profile.d" folder and 
```

Comment:
Lets omit this. I think it's too specific for the environment you're running Directus in (it wouldn't apply for containerized / platform as a service uses)

Reply:
done